### PR TITLE
WIP: Show parents on index and parent overview

### PR DESCRIPTION
**Pull request by Martchus**
Updated version of https://github.com/os-autoinst/openQA/pull/994

@coolo Removed the icons

I adapted the existing tests but still need to write tests for the new features.

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Main.pm [hunk 1]
```diff
@@ -22,21 +22,24 @@ use Date::Format;
...
 
 sub index {
     my ($self) = @_;
 
-    my $limit_builds    = $self->param('limit_builds')    // 3;
-    my $time_limit_days = $self->param('time_limit_days') // 14;
+    my $limit_builds = $self->param('limit_builds');
+    $limit_builds = 3 unless looks_like_number($limit_builds);
+    my $time_limit_days = $self->param('time_limit_days');
+    $time_limit_days = 14 unless looks_like_number($time_limit_days);
```
Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: these hides the fallback default values in code. Isn't there a nicer way?

Comment (Martchus) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: The main problem before was that the `some value // some default` syntax also uses `some value` if it is an empty string.

Don't know whether there's a better way. Defaults for index page and overview are even different so it wouldn't make sense to outsource it to a common place.

**Comment by coolo** [comment]
as disussed with Marius I will merge as is

**Comment by coolo** [comment]
(I guess I should have removed the WIP :)

**Pull request merged**